keyboards/xd68/xd68.c

```diff
@@ -0,0 +1,31 @@
+#include "xd68.h"
+#include "led.h"
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+	matrix_init_user();
+	led_init_ports();
+};
+
+void matrix_scan_kb(void) {
+	// put your looping keyboard code here
+	// runs every cycle (a lot)
+	matrix_scan_user();
+};
+
+void led_init_ports(void) {
+    // * Set our LED pins as output
+    DDRB |= (1<<2);
```

Comment:
```suggestion
    setPinOutput(B2);
    writePinHigh(B2);
```

The `writePinHigh()` is to prevent the LED flickering briefly on powerup.